format/src/main/thrift/carbondata.thrift

```diff
@@ -178,16 +178,27 @@ struct FileFooter{
     5: required list<BlockletIndex> blocklet_index_list;	// blocklet index of all blocklets in this file
     6: optional list<BlockletInfo> blocklet_info_list;	// Information about blocklets of all columns in this file
     7: optional list<BlockletInfo2> blocklet_info_list2;	// Information about blocklets of all columns in this file
-    8: optional list<BlockletInfo3> blocklet_info_list3;	// Information about blocklets of all columns in this file
-    9: optional dictionary.ColumnDictionaryChunk dictionary; // blocklet local dictionary
+    8: optional dictionary.ColumnDictionaryChunk dictionary; // blocklet local dictionary
+}
+
+/**
+* Footer for indexed carbon file
```

Comment:
I mean, can you mention in this comment that this struct is for V3 carbon file only